pkg/master/master.go

```diff
@@ -189,11 +189,6 @@ func (m *Master) InstallAPIs(c *Config) {
 			ParameterCodec:       api.ParameterCodec,
 			NegotiatedSerializer: api.Codecs,
 		}
-		if autoscalingGroupVersion := (unversioned.GroupVersion{Group: "autoscaling", Version: "v1"}); registered.IsEnabledVersion(autoscalingGroupVersion) {
```

Comment:
Are we removing the scale subresources? Not sure what's being proposed with this PR